src/android/CameraLauncher.java

```diff
@@ -26,11 +26,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
```

Comment:
I think these imports are not needed here. Are they leftover from an earlier iteration?